release-calendar/src/client/components/Calendar.tsx

```diff
@@ -37,7 +43,7 @@ export class Calendar extends React.Component<CalendarProps, {}> {
             right: 'dayGridMonth,timeGridWeek,listWeek',
           }}
           plugins={[dayGridPlugin, timeGridPlugin]}
-          eventSources={this.props.events}
+          eventSources={eventsToDisplay}
```

Comment:
Can we hide the time in the month view? The time could be useful in the week view.